scratch/PCA/extract_data.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
This should actually not be here. 
Since ‘main’ branch does not ignore *.ipynb however, and I branched from that, I could have forgotten this ignore line. 

Sorry about that, for reference if there is an .ipynb file I do not expect anyone else to jump into that. 

--
Henry




> On Aug 22, 2016, at 3:34 PM, Marc Daya <notifications@github.com> wrote:
> 
> In scratch/PCA/extract_data.ipynb <https://github.com/csm-adapt/tracr/pull/7#discussion_r75763517>:
> 
> > @@ -2,7 +2,7 @@
> >   "cells": [
> Is this file being intentionally committed, in spite of *.ipynb being added to the .gitignore?
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub <https://github.com/csm-adapt/tracr/pull/7/files/c17ee9cdcc76991b04da6460c9a3398b7897f8e9#r75763517>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AMZvXUKNuvOLwQxJ_5B8WRmLZG6qpJd3ks5qihX0gaJpZM4JqNtL>.
>